Explorer++/Explorer++/Storage.cpp

```diff
@@ -5,13 +5,38 @@
 #include "stdafx.h"
 #include "Storage.h"
 #include "../Helper/ProcessHelper.h"
+#include <wil/win32_helpers.h>
 #include <filesystem>
 
 namespace Storage
 {
 
+std::optional<std::wstring> GetExpandedEnvironmentVariable(const std::wstring &name)

```

Comment:
This function should be moved to one of the helper files. I'm not sure there's a particularly good place to put it, so perhaps just put in `Helper/Helper.cpp`.

A unit test for this method should also be added. There's some similar tests in [`ShellHelperTest.cpp`](https://github.com/derceg/explorerplusplus/blob/7e6b2b47005aaff01cde0dfffcc491b6b31f0282/Explorer%2B%2B/TestExplorer%2B%2B/ShellHelperTest.cpp#L33) that can be used as a reference.